core/src/main/java/io/aiven/kafka/tieredstorage/chunkmanager/DefaultChunkManager.java

```diff
@@ -62,7 +55,9 @@ public ChunkManager(final ObjectFetcher fetcher,
     public InputStream getChunk(final RemoteLogSegmentMetadata remoteLogSegmentMetadata, final SegmentManifest manifest,
                                 final int chunkId) throws StorageBackendException {
         final Chunk chunk = manifest.chunkIndex().chunks().get(chunkId);
-        final InputStream chunkContent = getChunkContent(remoteLogSegmentMetadata, chunk);
+        final String segmentKey = objectKey.key(remoteLogSegmentMetadata, ObjectKey.Suffix.LOG);
+
```

Comment:
nit: maybe place this new line after the fetcher call to separate the detransform logic

Reply:
Fixed